docs/projects.md

```diff
@@ -27,28 +27,28 @@ You can check all available options below.
 
 ```ts
 type ProjectConfigT = {
-  reactNativePath: ?string,
+  reactNativePath: ?string;
   project: {
-    android?: ProjectParamsAndroidT,
-    ios?: IOSProjectParams,
-    [key: string]: any,
-  },
-  assets: string[],
-  platforms: PlatformT,
+    android?: ProjectParamsAndroidT;
+    ios?: IOSProjectParams;
+    [key: string]: any;
+  };
+  assets: string[];
+  platforms: PlatformT;
   dependencies: {
     [key: string]: {
-      name: string,
-      root: string,
+      name: string;
+      root: string;
       platforms: {
-        [key: string]: PlatformSettingsT
-      },
-      assets: string[],
+        [key: string]: PlatformSettingsT;
+      };
+      assets: string[];
       hooks: {
-        [key: string]: string
-      }
-    },
-  },
-  commands: CommandT[]
+        [key: string]: string;
+      };
+    };
+  };
+  commands: CommandT[];
```

Comment:
Why did this happen? Different ESLint / TS versions? Which one should be correct?

CC: @thymikee

Reply:
We don't actively run prettier on MD files, so this happens when editor is configured to do so. This change is correct